.github/workflows/tests_linux.yml

```diff
@@ -52,45 +52,34 @@ jobs:
       - name: Build and run unit tests
         run: |
             cmake . -BBuild -G Ninja \
-              -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+              -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
Do we want these and below to be debug, or release with debug info?

Reply:
Building the C++ tests is the longest job, so I tried combining coverage and test assertions into one job. It appears `lcov` requires a `Debug` build.